pom.xml

```diff
@@ -4,26 +4,28 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>3.42</version>
-    <relativePath />
+    <version>3.43</version>
+    <relativePath/>
   </parent>
 
   <groupId>io.jenkins.plugins</groupId>
   <artifactId>warnings-ng</artifactId>
   <packaging>hpi</packaging>
   <name>Warnings Next Generation Plugin</name>
-  <version>5.0.1-SNAPSHOT</version>
+  <version>${revision}${changelist}</version>
   <url>https://wiki.jenkins.io/x/1YTeCQ</url>
   <description>Jenkins Warnings Next Generation plugin collects compiler warnings or issues reported by static
     analysis tools and visualizes the results. It has built-in support for numerous static analysis tools
     (including several compilers), see the list of supported report formats.
   </description>
 
   <properties>
+    <revision>5.1.0</revision>
```

Comment:
RevAPI points out among many other things that `QualityGateStatusHandler` is a new public class and so we need to bump the minor version (although I guess we could make the class package-private if desired).